src/main/resources/db/migration/V46__alter_video_link_appointment.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE VIDEO_LINK_APPOINTMENT ADD (location_id BIGINT, start_date_time TIMESTAMP, end_date_time TIMESTAMP);
```

Comment:
we'll be wanting to do queries on start and end time and prison id - will need indexes on these?